rpc/net.go

```diff
@@ -72,6 +74,10 @@ func (s *NetService) Peers(_ *http.Request, _ *interface{}, resp *rpctypes.Peers
 
 // QueryAll discovers peers who provide a certain coin and queries all of them for their current offers.
 func (s *NetService) QueryAll(_ *http.Request, req *rpctypes.QueryAllRequest, resp *rpctypes.QueryAllResponse) error {
+	if s.isBootnode {
+		return errUnsupportedForBootnode
+	}
```

Comment:
Would it be worth leaving this function in for troubleshooting? It don't see any dependencies on a local maker or taker.

Reply:
the main issue is I didn't want the bootnode to support any stream protocols since they aren't necessary to its function, I could add the query protocol in if you think it's useful

Reply:
I'm imagining scenarios where a swapd node can't connect to a bootnode, or 2 peers don't see each other's offers, or a swapd instance is not finding any relayers despite relayers existing. I can at least imagine scenarios where being able to query the state of the DHT on the bootnode and maybe even reachability of peers could aid in diagnosing where/how the problem is happening. We can always ticket it and add it later if you think it will be useful but don't want to take it on now.

Reply:
I think discover should be fine for now since that shows who's discoverable on the DHT, we can add query in later if necessary